src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {

```

Comment:
Interface names should be nouns or noun phrases, not verbs. Instead of `FigureDraw`, a name like `Drawable` or `DrawableFigure` would be more appropriate as per the checklist item: Don't use verbs for class/interface names.